### PR TITLE
Add tests for empty data search

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
Add cypress End-To-End tests for:

1. Search without entering any city and country

2. Search with a valid city name but an empty value in country

## Related Issue:
- https://github.com/bartius-nigel/weather-comparitor/issues/13

### CODE REVIEW

File: cypress/integration/examples/searchCity.spec.js [hunk 1]
```diff
@@ -8,26 +9,48 @@ context('Weather', () => {
...
+            cy.on('window:alert', (str) => {
+                try {
+                 expect(str).to.equal(popOpMessages.invalidData)
+                 } catch (err) {
+                    return reject(err)
+                }
+                resolve();
+            })
+        }))    
+   })
```
Comment (Talank) [inline cypress/integration/examples/searchCity.spec.js:54]: Looks like you need to fix the indentation

File: cypress/support/commands.js [hunk 2]
```diff
@@ -14,3 +26,11 @@ Cypress.Commands.add('assertTableRowNumbers', (expectedNumber) => {
         .its('length')
         .should('eq',expectedNumber)
 })
+
+Cypress.Commands.add('assertPopOpIsDisplayed', (displayedMessage) => {
+    cy.log(displayedMessage)
```
Comment (Talank) [inline cypress/support/commands.js:31]: looks like a debug step :thinking:

File: cypress/support/commands.js [hunk 3]
```diff
@@ -14,3 +26,11 @@ Cypress.Commands.add('assertTableRowNumbers', (expectedNumber) => {
...
+        cy.on('window:alert', (str) => {
+            cy.log(str)
```
Comment (Talank) [inline cypress/support/commands.js:33]: remove the log

Comment (jasson99) [reply cypress/support/commands.js]: fixed

File: cypress/support/commands.js (hunk 2 above)
Comment (jasson99) [reply cypress/support/commands.js]: fixed

File: cypress/integration/examples/searchCity.spec.js (hunk 1 above)
Comment (jasson99) [reply cypress/integration/examples/searchCity.spec.js]: fixed

**Comment by bartius-nigel** [comment]
This looks really good, but there are some changes here that are similar to the other MRs up on this repo, so I will wait until those are merged before reviewing this. Thank you for the contribution!!

**Comment by jasson99** [comment]
@bartius-dev
